MIDIchlorians/MIDIchlorians/Assets.xcassets/yellowButton.imageset/Contents.json

```diff
@@ -4,6 +4,14 @@
       "idiom" : "universal",
```

Comment:
I have no clue where these came from. I didn't change any of this. Maybe when I pulled master? But why would they show up here then?